portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -30,10 +33,21 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  private List<String> demoComments = new ArrayList<String>();
+  List<String> demoComments = new ArrayList<String>();
```

Comment:
I don't think you need it anymore, as the comments are no longer in memory.

Reply:
Deleted, I forgot to do so.